doc/workflows_and_tools/tools/index.rst

```diff
@@ -31,6 +31,303 @@ a glimpse in the connection to your user store, you could as well in a first
+
+tokenattribute-value-greater-than INTEGER
+.........................................
+
+Match if the value of the token attribute is greater than the given value.
+
+Example::
+
+    privacyidea-token-janitor find --tokenattribute active --tokenattribute-value-greater-than 0
+
+Search for all tokens with the tokenattribute-key ``active`` and the associated tokenattribute-value greater than ``0``.
+
+Tokeninfo-key
+*************
+
+The tokeninfo-key to match.
+
+There are different ways of filtering here.
+
+has-tokeninfo-key
+.................
+
+Filters for tokens that have given the specified tokeninfo-key.
+
+Example::
+
+    privacyidea-token-janitor find --has-tokeninfo-key import_time
+
+Searches for all tokens that have stored the tokeninfo-key ``import_time``.
+
+**Note, that it is not important, what value the "import_time" actually has!**
+
+has-not-tokeninfo-key
+.....................
+
+Filters for tokens that have not given the specified tokeninfo-key.
+
+Example::
+
+    privacyidea-token-janitor find --has-not-tokeninfo-key import_time
+
+Searches for all tokens that didnt stored the tokeninfo-key ``import_time``.
+
+tokeninfo-value REGEX|INTEGER
+.............................
+
+The tokeninfo-value to match.
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key tokenkind --tokeninfo-value software
+
+Search for all tokens with the tokeninfo-key ``tokenkind`` and the associated tokeninfo-value ``software``.
+
+tokeninfo-value-less-than INTEGER
+.................................
+
+Interpret tokeninfo-values as integers and match only if they are smaller than the given integer.
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key timeWindow --tokeninfo-value-less-than 200
+
+Search for all tokens with the tokeninfo-key ``timeWindow`` and the associated tokeninfo-value below ``200``.
+
+tokeninfo-value-greater-than INTEGER
+....................................
+
+Interpret tokeninfo-values as integers and match only if they are greater than the given integer.
+
+Example::
+
+    privacyidea-token-janitor find --tokeninfo-key timeWindow --tokeninfo-value-greater-than 100
+
+Search for all tokens with the tokeninfo-key ``timeWindow`` and the associated tokeninfo-value greater than ``100``.
+
+Actions
+*******
+
+Actions that can be performed on the found tokens.
+
+mark - disable - delete - unassign - export - listuser - tokenrealms
+
+mark
+....
+
+**Mark** makes it possible to mark single or multiple tokens in order to carry out further actions with them later.
+
+Please note that without a previous selection of a certain token, all found tokens are automatically marked.
+
+Example::
+
+    privacyidea-token-janitor find --serial OATH0004C934 --action mark
+
+The token with the serial ``OAUTH0004C934`` will be marked.
+
+
+disable
+.......
+
+With **disable** single or multiple tokens can be disabled.
+
+Please note that without a previous selection of a certain token, all found tokens will be disabled.
```

Comment:
Honestly, I find this wording still a bit unclear.

The token janitor can be used to find tokens based on filters. If not filters are applied, all tokens will be found.
Actions are performed by the token janitor on all found tokens.

So your sentence is a bit unlogical. Because "all found tokens will be disabled" is **always** true, not only without a previous selection. Do you understand my logical nitpicking?